java/src/jmri/jmrit/logixng/tools/swing/LogixNGSwingBundle.properties

```diff
@@ -108,6 +108,10 @@ may be others not known to the import tool.                                 \
 </table></html>
 
 Import_ButtonImport         = Import
+TitleLogixsImportSuccess    = The import was successful
+LogixsAreImported           = The Logixs are imported
```

Comment:
Should this be plural?  This PR is about importing one Logix, right?

Reply:
Yes. The changes to `LogixTableAction` is about exporting one Logix to LogixNG. But the changes to `ImportLogixFrame` is about importing all the Logixs to LogixNG. `LogixTableAction` is related to Logix, so therefore it's exporting. But `ImportLogixFrame` is a LogixNG tool, so therefore it's importing.

While working on `LogixTableAction`, I also took the time to localize `ImportLogixFrame`. I hadn't localized `ImportLogixFrame` before.